root/etc/e-smith/events/actions/interface-rename

```diff
@@ -32,6 +32,11 @@ if ($argnum == 0) {
     exit 0;
 }
 
+if (defined($ARGV[0]) && $ARGV[0] eq '--skip-network') {
```

Comment:
Usage of `grep` is not required here because, if present, skip-network is the first positional parameter.